haystack/preprocessor/queryexpander.py

```diff
@@ -0,0 +1,90 @@
+from langdetect import detect
+import spacy
+from spacy_wordnet.wordnet_annotator import WordnetAnnotator 
+
+
+
+class QueryExpander:
+    def __init__(self, num_keyword_repeats=3, num_synonyms_per_word=3):
```

Comment:
let's add an arg `domains=None` here  so that users can supply a list of "economy domains"

Reply:
done